### PR TITLE
Feature/sciapp pages

### PR DESCRIPTION
**Pull request by eheinrich**
- Add a viewset for science applications API endpoints, replacing the forms that had been used.
- Combined PDF generation has been moved entirely to the new frontend and removed from here.
- Remove all the old views that provided science application pages. Since these are the final set of pages that are being replaced, also removed other remaining frontend code and configuration.

### CODE REVIEW

File: observation_portal/common/configdb.py [hunk 1]
```diff
@@ -312,21 +312,21 @@ def get_instrument_names(
                 instrument_names.add(instrument['code'].lower())
         return instrument_names
 
-    def get_instrument_types_per_telescope_name(self, exclude_states=None) -> dict:
-        """Get a set of instrument types.
+    def get_telescope_name_by_instrument_types(self, exclude_states=None) -> dict:
+        """Get a mapping of instrument type to telescope name.
 
-        Instrument types are returned by telescope name (1 meter, 2 meter, etc...)
+        Telescope names the are returned are e.g. 1 meter, 2 meter, etc...
```
Comment (jnation3406) [inline observation_portal/common/configdb.py:318]: that instead of the?

File: observation_portal/proposals/models.py [hunk 2]
```diff
@@ -10,7 +10,9 @@
 from collections import namedtuple
 import logging
 
+
```
Comment (jnation3406) [inline observation_portal/proposals/models.py:13]: is this second blank line intentional?

**Review by jnation3406 — APPROVED** [review]

File: observation_portal/proposals/models.py (hunk 2 above)
Comment (eheinrich) [reply observation_portal/proposals/models.py]: Nope, removed it

File: observation_portal/common/configdb.py (hunk 1 above)
Comment (eheinrich) [reply observation_portal/common/configdb.py]: Done

**Pull request merged**